themes/garland/maintenance-page.tpl.php

```diff
@@ -22,7 +22,7 @@
       <?php print phptemplate_get_ie_styles(); ?>
     <![endif]-->
   </head>
-  <body<?php print phptemplate_body_class($left, $right); ?>>
+  <body<?php phptemplate_body_class($left, $right); ?>>
```

Comment:
Hmhmhm. This is unfortunate. 

Indeed the print statement does not do anything, because phptemplate_body_class() as the _only_ function in template.php has its own print statement. (I'm guessing that was a bug / oversight.)

So I can understand it messes with your static code analysis or IDE.

But wouldn't removing the print statement from here, cause more confusion for human code/template readers, who don't know that phptemplate_body_class() is quasi-buggy?